train.py

```diff
@@ -500,7 +500,9 @@ def main():
         from diffusers import FlowMatchEulerDiscreteScheduler
 
         noise_scheduler = FlowMatchEulerDiscreteScheduler.from_pretrained(
-            args.pretrained_model_name_or_path, subfolder="scheduler"
+            args.pretrained_model_name_or_path,
+            subfolder="scheduler",
+            shift=1 if args.flux else 3,
```

Comment:
well this would be if we use timestep_scheme=sd3 for flux. but we don't. so this does nothing